### DISCUSSION TITLE
Usage example with Timber theme

### DISCUSSION THREAD

**Discussion by jasalt** [TCattd/HTMX-API-WP#12] [Ideas]
Timber is a nice MVC style base theme using PHP Twig templating language (similar to Django, Laravel Blade etc). I'm currently experimenting how to use it with HTMX and this plugin with it's starter-theme https://github.com/jasalt/starter-theme/blob/2.x-htmx-test/README.md.

I think Timber could be a good match with HTMX that has been seen to sit well with MVC web application frameworks and patterns there could be followed.

**Comment by EstebanForge** [comment]
Nice to know about this. Thanks @jasalt :)

Do you mind if i put a link to that theme into this repository readme? beside the simple demo theme i made.

**Reply by jasalt** [reply]
Please do, I try keep it up to date as I continue experimenting..

**Comment by proweb** [comment]
Thanks
